warehouse/templates/packaging/detail.html

```diff
@@ -21,6 +21,8 @@
    {% set icon = "fas fa-cloud-download-alt" %}
 {% elif name|lower in ["home", "homepage", "home page"] %}
   {% set icon = "fas fa-home" %}
+{% elif name|lower in ["changelog", "change log", "changes", "release notes", "news", "what's new"] %}
```

Comment:
@scop I often use category prefixes for links. See https://pypi.org/project/aiohttp/. So it'd be better to match if any of these are in the name and not just match exactly. In this case, I'd rather use `Docs: changelog` to follow the same style as `Docs: RTD`.

Reply:
I'm sure the warehouse maintainers would consider such a PR :)